x/namecoin/handler.go

```diff
@@ -170,7 +170,7 @@ func (h SetNameHandler) Deliver(ctx weave.Context, db weave.KVStore,
 		return res, err
 	}
 	if obj == nil {
-		return res, ErrNoSuchWallet(msg.Address)
+		return res, errors.ErrNotFound.Newf("wallet %X", msg.Address)
```

Comment:
Careful to use %X with weave.Address....

Since address.String will output as hex, using %X will hex encode the hex output and create confusion.

I just caught this one case, but please check the use of Address variables elsewhere also

Reply:
@ethanfrey this is the same behavior we had before refactor. I can use string %s if that's easier.

Reply:
actually, this is used all over the place in our code. I will change address usages to %s